mParticle-Apptentive/MPKitApptentive.m

```diff
@@ -270,10 +268,14 @@ - (nonnull MPKitExecStatus *)logBaseEvent:(nonnull MPBaseEvent *)event {
 
 - (MPKitExecStatus *)routeEvent:(MPEvent *)event {
     NSDictionary *eventValues = event.customAttributes;
+
+    UIViewController * rootViewController = [[[[UIApplication sharedApplication] delegate] window] rootViewController];
```

Comment:
I might be missing something, but this case should already be handled by the Apptentive SDK. 

Passing `nil` results in a nil `presentingViewController` property on the `InteractionPresenter` instance, which will walk up the view controller stack of the application's `keyWindow`. It also handles the (surprisingly common) case where an interaction tries to present on a view controller that is being dismissed.